lib/diagnostic.rb

```diff
@@ -78,12 +97,15 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a HouseCat, and I go meow"
 
 ## Explain why this would be the output, based on the method lookup chain.
 # your response as a comment here
 ##
-
+the say_hello pulls from the class animal.  since the housecat is the chid
+of the cat is the chid of anial it will put the string of "I am a etc etc"
+{self.class.name} in the string will use the housecat and the sound will use
+the @sound for house cat = string for meow
 ##
 # Define a new class, 'Lion', which (a) inherits from 'Cat',
```

Comment:
In direct inheritance, you're limited to inheriting from a single source.
In composition, you can draw from as many modules as you like.